.gitignore

```diff
@@ -4,3 +4,4 @@
 /data/hypha.xml
 /data/digest
 /data/hypha.stats
+/mockup/node_modules
```

Comment:
This commit adds third-party sources. It would be good if the commit message lists the upstream locations and versions of these.

Reply:
This actually ignores third-party sources.

Reply:
I meant the commit this change is in, not this particular change (I just took the first change in the commit, since you can't comment on a commit anymore in github). However, I see now that this is not really clear.

In any case, I meant files like normalize.css. I've now found the source for this file and put it in the commit message. I've also removed chota.css and cclogo.png, both of which seemed unused (and chota.css was in data/ rather than data/css, which seemed odd).

For the dosis font, I couldn't find a canonical source. Perhaps we should clarify the source along with the license (see #89).